### ISSUE TITLE
load problems

### ISSUE THREAD

**Issue by rimusz** [AntonioMeireles/coreos-xhyve#13]
```
[corectl] beta/877.1.0 already available on your system
[corectl] unable to guess the MAC Address from the provided UUID (4420CA07-3C45-4110-86D7-D999D9FB6E2E). Using a randomly generated one one
Error: Aborting. Another VM is running with same name. (core-01)
```

```
$ ./corectl ps
[corectl] found 0 running VMs, summing 0 vCPUs and 0MB in use.
```

```
$ cat dhcpd_leases
{
    ip_address=192.168.64.2
    hw_address=1,e6:16:63:24:cf:32
    identifier=1,e6:16:63:24:cf:32
    lease=0x56634ffe
}
```

**Comment by AntonioMeireles** [comment]
i believe that you just had hit the issue fixed by #11 (consumed in v[0.0.11](https://github.com/TheNewNormal/corectl/releases/tag/v0.0.11)). anyway leaving this open a few more days in case i'm wrong and you hit this again.

**Comment by rimusz** [comment]
it looks it is fixed, we can close this one
